src/OpenTelemetry.Contrib.PersistentStorage/LocalFileBlob.cs

```diff
@@ -0,0 +1,120 @@
+﻿// <copyright file="LocalFileBlob.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.IO;
+
+namespace OpenTelemetry.Contrib.PersistentStorage
+{
+    /// <summary>
+    /// The <see cref="LocalFileBlob"/> allows to save a blob
```

Comment:
Minor, maybe consider `FileStorageBlob` or `FileBlob`. The fact that the implementation is using file API doesn't necessarily require the file to be local (e.g. it could be UNC path, or a local path backed by a remotely shared folder).